compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -173,8 +201,11 @@ static bool isFullyDynamicOp(linalg::LinalgOp op) {
 /// given LinalgOp, depending on the op traits and the target architecture.
 static VectorPreProcStrategy
 getVectorPreProcStrategy(linalg::LinalgOp linalgOp) {
-  // Generic strategies.
+  // 1. Select a strategy based on the input from a user.
+  if (preprocessStrategyFromCL != VectorPreProcStrategy::Heuristics)
+    return preprocessStrategyFromCL;
 
+  // 2. Select a strategy based on heuristics.
   if (linalgOp.hasBufferSemantics()) {
     return VectorPreProcStrategy::None;
   }
```

Comment:
Can we remove `1.` and `2.` from the comment? I think it is good for algorithm explanation in some transforms. The logic here is quite simple, so we don't need to itemize them. Perhaps we can make the first comment be "Use `preprocessStrategyFromCL` if it is set by a user". What do you think?

Also we prefer wrapping single statement with braces in IREE. IREE follows mixed LLVM/Google style guide, and this is one of few difference.

Reply:
Thanks, happy to make the updates!

> Perhaps we can make the first comment be "Use preprocessStrategyFromCL if it is set by a user". What do you think?

I'd rather avoid repeating the variable name in a comment (when renaming a variable some people use their IDEs that only update code references and skip comments). How about this:
> If set, use the strategy selected by a user.
?